### DISCUSSION TITLE
What's the bevy release plan?

### DISCUSSION THREAD

**Discussion by rosymati** [bevyengine/bevy#2474] [Q&A] [answered]
When can we expect a new bevy version to be released and with which criteria? There have already been quite a few changes from 0.3.0 but there's clearly a reason for the wait otherwise we would already have a 0.4.0.
Would it make sense to add milestones so that it's easy to track what the blocking issues for a version to be released? 
Right now the only project is the roadmap which doesn't really help outside of getting a general view of the project goals.
Marking an issue with a 0.4.0 (and future versions) milestone would really help users and contributors get a better understading of the project status

**Comment by alice-i-cecile** [comment, marked as answer]
0.4 is now live, but clearer planning on the roadmap, timelines and feature inclusion would be appreciated going forward :) This becomes particularly true as the project matures, and releases need to be properly tested, documented and synced with the surrounding ecosystem.
